index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world</h1>
```

Comment:
```suggestion
<!DOCTYPE html>
<html>
  <head>
    <title>Page Title</title>
  </head>
  
  <body>
    <h1>Hello world</h1>
  </body>
</html>
```

Reply:
In the manual I have seen that I need to write in index.html only 1 row: <h1>Hello world!</h1>